.github/workflows/ci.yml

```diff
@@ -19,9 +19,11 @@ jobs:
       # All features
       - run: cargo check --all-targets --all-features
       - run: cargo test --all-features
-      # No default features
-      - run: cargo check --all-targets --no-default-features
-      - run: cargo test --no-default-features
+      # No default features. Only works on Rust 1.81
```

Comment:
Probably just better to have 2 separate jobs here and let it install 1.81 in the `no_std` job. It'll be fast anyway since it gets the sparse support from Cargo.